newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -95,16 +98,24 @@ const ExtensionInstallDialog = ({
       if (canInstallExtension) {
         if (alreadyInstalled) {
           const answer = Window.showConfirmDialog(
-            'This extension is already in your project, this will install the latest version. You may have to do some adaptations to make sure your game still works. Do you want to continue?'
+            (extensionUpdate && extensionUpdate.type) === 'major'
+              ? i18n._(
+                  t`This extension update contains a breaking change. You will have to do some adaptations to make sure your game still works. We advise to back up your game before proceeding. Do you want to continue?`
```

Comment:
> You will have to do some adaptations to make sure your game still works.
It's a nitpick but for me "make sure" mean "verify" and doing changes is more than verifying.
I guess it should be either:
`Make sure your game still works and do required adaptations.`
`Adaptations may be required for your game to keep working.`